app/src/shared/lib/hooks/useIsToggle.ts

```diff
@@ -0,0 +1,15 @@
+import { useCallback, useState } from 'react';
+
+export const useIsToggle = (initialState?: boolean) => {
```

Comment:
```suggestion
export const useToggle = (initialState?: boolean) => {
```

Reply:
toggle - просто переключатель / noun

или
```suggestion
export const useIsToggled = (initialState?: boolean) => {
```